packages/server/lib/browsers/chrome.ts

```diff
@@ -329,9 +315,9 @@ const _handleDownloads = async function (client, dir, automation) {
   })
 }
 
-const _setAutomation = (client, automation) => {
+const _setAutomation = async (browserCriClient: BrowserCriClient, client: CRIWrapper.Client, automation) => {
```

Comment:
Could we slim to something like:
```suggestion
const _setAutomation = async (client: CRIWrapper.Client, automation, closeCurrentTarget: () => {}) => {
```

Reply:
Updated here: https://github.com/cypress-io/cypress/commit/1efb6cef093acca9c1341e61b84c799666d81965